src/tools/script_tools.cpp

```diff
@@ -285,6 +285,16 @@ void RegisterScriptTools(ToolManager* toolMgr)
 	SmartPtr<luashell::LuaShell> shell = make_sp(new luashell::LuaShell());
 	ParseDirAndCreateTools(toolMgr, QDir(":/scripts"), "Scripts", shell);
 	ParseDirAndCreateTools(toolMgr, app::UserScriptDir(), "Scripts", shell);
+
+	if(app::UserDataDir().exists("custom_user_script_dirs")){
```

Comment:
This check is not really necessary. And the location where the customUserScriptDirs are stored is actually an implementation detail of `app::CustomUserScriptDirs()`, I think.